README.md

```diff
@@ -449,6 +449,8 @@ All these projects contain steps and resources required for reproduction.
 - [Automatic Plant Watering System Using a micro:bit](https://www.instructables.com/Automatic-Plant-Watering-System-Using-a-Microbit/) - How to build an automatic plant watering system using a micro:bit, moisture sensor, and some other small electronic components.
 - [Sorting the Object Using Ferb Detective Agent](https://www.instructables.com/Sorting-the-Object-Using-Ferb-Detective-Agent/) - How to design a DIY colour sorter that can sort black and white objects using an IR sensor.
 - [micro:bit Solar Tracker](https://www.instructables.com/Microbit-Solar-Tracker/) - How to build a solar tracker with a micro:bit, light sensors and servos.
+- [Mewt](https://github.com/dakota-mewt/mewt/blob/main/README_Microbit.md) - Single button press to mute/unmute all system microphones, shows you status and when the microphone is actively broadcasting.
```

Comment:
Thanks for the submission!
I think the Projects section is more for projects for readers to be able to replicate (or use for inspiration), but this is more or a tool that is ready to use?
If that's the case I'd put this in the Miscellaneous section.

Reply:
I guess you can consider it that.  I was able to utilize the built-in Microbit v2 inputs/sensors so that there's nothing else needed to be done hardware-wise.  For Windows, I also packaged and automated the hex upload along with the PC-side code.  

But for Linux/Mac usage, it does require code to be loaded manually on the system side and on the Microbit side.  

I trust that you know which section is best.  I'll cancel this PR and resubmit at for Miscellaneous